configuration-wire/precomputed-obfuscated.json

```diff
@@ -0,0 +1,21 @@
+{
+  "version": 1,
+  "precomputed": {
+    "subjectKey": "test-subject-key",
+    "subjectAttributes": {
+      "device": "iPhone",
+      "country": "USA"
```

Comment:
Note: these are not obfuscated because the `obfuscated` boolean is not an attribute at this level of the object